examples/deferred_processing/automated_deferred_processing/branch-mover-no-auth.json

```diff
@@ -0,0 +1,22 @@
+{
+  "pipeline": {
+    "name": "branch-mover"
+  },
+  "input": {
+    "cron": {
+        "name": "tick",
+        "spec": "@every 1m",
+        "overwrite": true
+    }
+  },
+  "transform": {
+      "cmd": ["sh" ],
+      "stdin": [
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+          "pachctl create branch edges_dp@master --head dev"
+      ],
+    "image": "pachyderm/pachctl:1.11.0"
```

Comment:
oh nice, I didn't realize `pachyderm/pachctl` would be helpful for deferred processing